src/app-bridge/app-bridge-state.ts

```diff
@@ -10,6 +11,17 @@ export type AppBridgeState = {
   theme: ThemeType;
   locale: LocaleCode;
   saleorApiUrl: string;
+  user?: {
+    /**
+     * Original permissions of the user that is using the app.
+     * *Not* the same permissions as the app itself.
+     *
+     * Can be used by app to check if user is authorized to perform
+     * domain specific actions
+     */
+    ownPermissions: Permission[];
```

Comment:
```suggestion
    permissions: Permission[];
```

If the permissions are inside the user object, prefix `own` IMO can be skipped.

Can we get access to the permissions of the app in the root level of the appBridge?

Reply:
Alternative approach - what if we expose the contents of the token? We'll be able to avoid mapping one object to another, which ould lead to confusion 

```
{
  "iat": 1669191845,
  "owner": "saleor",
  "iss": "demo.eu.saleor.cloud",
  "exp": 1669278245,
  "token": "sblFekZuBIGW",
  "email": "admin@example.com",
  "type": "thirdparty",
  "user_id": "VXNlcjoyMg==",
  "is_staff": true,
  "app": "QXBwOjI3NQ==",
  "permissions": [],
  "user_permissions": [
    "MANAGE_PAGE_TYPES_AND_ATTRIBUTES",
    "MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES",
    "MANAGE_DISCOUNTS",
    "MANAGE_TRANSLATIONS",
    "MANAGE_CHECKOUTS",
    "HANDLE_CHECKOUTS",
    "MANAGE_ORDERS"
  ]
}
```

Reply:
I've changed to the suggested `permissions`.

We can add token too in the next PRs